guvnor-m2repo-editor/guvnor-m2repo-editor-client/src/main/java/org/guvnor/m2repo/client/upload/UploadFormViewImpl.java

```diff
@@ -195,8 +146,80 @@ private String getWebContext() {
         return context;
     }
 
-    protected void showUploadingBusy() {
-        // LoadingPopup.showMessage( "Uploading...");
+    public void showUploadingBusy() {
+        // TODO: "Uploading..." needs localization
+        BusyPopup.showMessage("Uploading...");
```

Comment:
Have you verified this works OK and does not affect the UI at runtime (or do you rely on the tests passing meaning everything is good?)... I just ask as I have a nagging memory that showing the ```BusyPopup``` affected the operation of the upload (e.g. it caused it not to work)... I could be wrong.. I just have this nagging recollection.